app/views/passengers/show.haml

```diff
@@ -11,6 +11,14 @@
 %p
   %b Phone:
   = @passenger.phone
+%p
+  %b Subscribed to SMS:
+  = @passenger.subscribed_to_sms ? "Yes" : "No"
+
+- if @passenger.subscribed_to_sms?
+  %p
+    %b SMS address:
+    = @passenger.phone.delete('-') + @passenger.carrier.gateway_address
```

Comment:
What if the phone number is saved as `(413).666.6666`, will this still work?
P.S. Please don't validate phone number formats, that's not what I'm getting at. Just change this line.

Reply:
I assumed phone numbers were already validated for some reason, I'll fix it.